src/plugin/env-api-client-v3.js

```
@@ -172,21 +229,20 @@ class EnvApiClient {
               });
           } else {
             logger.error(
-              `Fallback not supported and/or wrong error code: ${err.statusCode ||
-                err}`
+              `Fallback not supported and/or wrong error code ${err.statusCode}: ${err.message}`
             );
-            throw err;
+            throw err.message;
```

why not just re-throw the same error?

---

because the error is not a proper `new Error({name : "", message:""})` , it is a custom obj that has a `statusCode` property https://github.com/InVisionApp/kit-deploymentizer/pull/90/files#diff-7edbf28adc2435bdc3c872a3ac715cc0R151

---

ah ok, in that case is there a reason not to create an error object? 
```javascript
throw new Error(err.message)
```

---

done